src/main/resources/com/atlassian/jira/cloud/jenkins/configuration/config.jelly

```diff
@@ -1,8 +1,158 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" >
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
```

Comment:
the Main Jelly tag!!! declares a bunch of tags values i use further down
j Jelly
l Layout
f Form
c Credentials